src/style.css

```
@@ -0,0 +1,104 @@
+body {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.container {
+  display: flex;
+  flex-wrap: wrap;
+  flex-direction: column;
+  justify-content: space-between;
+  align-content: stretch;
+  align-items: baseline;
+  line-height: 300px;
+  text-align: center;
+  font-family: "Arial", "serif";
+  font-size: 100px;
+  color: white;
```

```suggestion
  color: #fff;
```
FYI: default color is white, so you no need it